<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="SmoothScrollArea" name="SmoothScrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>959</width>
        <height>720</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="SimpleCardWidget" name="SimpleCardWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>15</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item>
           <widget class="IconWidget" name="IconWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>84</width>
              <height>84</height>
             </size>
            </property>
            <property name="icon">
             <iconset resource="../resource.qrc">
              <normaloff>:/icons/images/icons/database_export.svg</normaloff>:/icons/images/icons/database_export.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="VerticalSeparator" name="VerticalSeparator"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="LargeTitleLabel" name="LargeTitleLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>入库</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="BodyLabel" name="BodyLabel_7">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>将商品录入系统</string>
              </property>
              <property name="lightColor" stdset="0">
               <color>
                <red>96</red>
                <green>96</green>
                <blue>96</blue>
               </color>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="CardWidget" name="CardWidget">
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>15</number>
              </property>
              <property name="leftMargin">
               <number>15</number>
              </property>
              <property name="topMargin">
               <number>15</number>
              </property>
              <property name="rightMargin">
               <number>15</number>
              </property>
              <property name="bottomMargin">
               <number>15</number>
              </property>
              <item>
               <widget class="SubtitleLabel" name="SubtitleLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>入库信息</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="HorizontalSeparator" name="HorizontalSeparator">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <property name="leftMargin">
                 <number>15</number>
                </property>
                <property name="topMargin">
                 <number>15</number>
                </property>
                <property name="rightMargin">
                 <number>15</number>
                </property>
                <property name="bottomMargin">
                 <number>15</number>
                </property>
                <property name="spacing">
                 <number>15</number>
                </property>
                <item row="4" column="1">
                 <widget class="SpinBox" name="SpinBox_2">
                  <property name="toolTip">
                   <string>进货批次,可以填写当月的过往批次,但是不允许批次数超过最新批次数+1</string>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>999</number>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="SpinBox" name="SpinBox">
                  <property name="toolTip">
                   <string>同样的商品的数量,注意该数量不要超过打印机的打印纸数量</string>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>9999</number>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="BodyLabel" name="BodyLabel_4">
                  <property name="text">
                   <string>商品数量</string>
                  </property>
                  <property name="buddy">
                   <cstring>SpinBox</cstring>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="LineEdit" name="LineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="BodyLabel" name="BodyLabel_5">
                  <property name="text">
                   <string>批次</string>
                  </property>
                  <property name="buddy">
                   <cstring>SpinBox_2</cstring>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="SwitchButton" name="SwitchButton">
                  <property name="toolTip">
                   <string>开启后会将这一批货作为一批新的批次</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <property name="onText">
                   <string>开启</string>
                  </property>
                  <property name="offText">
                   <string>关闭</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="BodyLabel" name="BodyLabel">
                  <property name="text">
                   <string>衣服名称</string>
                  </property>
                  <property name="buddy">
                   <cstring>LineEdit</cstring>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="BodyLabel" name="BodyLabel_6">
                  <property name="text">
                   <string>自动批次</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="LineEdit" name="LineEdit_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0" colspan="2">
                 <widget class="PrimaryPushButton" name="PrimaryPushButton">
                  <property name="toolTip">
                   <string>提交数据到右侧表格内</string>
                  </property>
                  <property name="text">
                   <string>确认</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="BodyLabel" name="BodyLabel_3">
                  <property name="text">
                   <string>价格</string>
                  </property>
                  <property name="buddy">
                   <cstring>DoubleSpinBox</cstring>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="BodyLabel" name="BodyLabel_2">
                  <property name="text">
                   <string>品牌</string>
                  </property>
                  <property name="buddy">
                   <cstring>LineEdit_2</cstring>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="DoubleSpinBox" name="DoubleSpinBox">
                  <property name="toolTip">
                   <string>支持小数点后两位,最多支持单价为十万</string>
                  </property>
                  <property name="maximum">
                   <double>100000.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="TransparentPushButton" name="TransparentPushButton_4">
               <property name="text">
                <string>删除当前行</string>
               </property>
               <property name="icon">
                <iconset resource="../resource.qrc">
                 <normaloff>:/icons/images/icons/delete.svg</normaloff>:/icons/images/icons/delete.svg</iconset>
               </property>
               <property name="hasIcon" stdset="0">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="TransparentPushButton" name="TransparentPushButton">
               <property name="text">
                <string>清空全部</string>
               </property>
               <property name="icon">
                <iconset resource="../resource.qrc">
                 <normaloff>:/icons/images/icons/erase.svg</normaloff>:/icons/images/icons/erase.svg</iconset>
               </property>
               <property name="hasIcon" stdset="0">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="TransparentPushButton" name="TransparentPushButton_3">
               <property name="toolTip">
                <string>数据会先打印再入库,确保打印机内有充足打印纸</string>
               </property>
               <property name="text">
                <string>入库</string>
               </property>
               <property name="icon">
                <iconset resource="../resource.qrc">
                 <normaloff>:/icons/images/icons/sign_out.svg</normaloff>:/icons/images/icons/sign_out.svg</iconset>
               </property>
               <property name="hasIcon" stdset="0">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="TableWidget" name="TableWidget_2">
             <property name="showGrid">
              <bool>false</bool>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="rowCount">
              <number>20</number>
             </property>
             <property name="selectRightClickedRow">
              <bool>false</bool>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <row>
              <property name="text">
               <string>1</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>2</string>
              </property>
             </row>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>商品名称</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>品牌</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>价格</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>批次</string>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string>Nike运动鞋</string>
              </property>
             </item>
             <item row="0" column="1">
              <property name="text">
               <string>NIKE</string>
              </property>
             </item>
             <item row="0" column="2">
              <property name="text">
               <string>367</string>
              </property>
             </item>
             <item row="0" column="3">
              <property name="text">
               <string>202401001</string>
              </property>
             </item>
             <item row="1" column="0">
              <property name="text">
               <string>Nike运动鞋上衣</string>
              </property>
             </item>
             <item row="1" column="1">
              <property name="text">
               <string>NIKE</string>
              </property>
             </item>
             <item row="1" column="2">
              <property name="text">
               <string>246</string>
              </property>
             </item>
             <item row="1" column="3">
              <property name="text">
               <string>202401002</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TransparentPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>IconWidget</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HorizontalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>VerticalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>CardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SmoothScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LargeTitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LineEdit</tabstop>
  <tabstop>LineEdit_2</tabstop>
  <tabstop>DoubleSpinBox</tabstop>
  <tabstop>SpinBox_2</tabstop>
  <tabstop>SpinBox</tabstop>
  <tabstop>PrimaryPushButton</tabstop>
  <tabstop>TransparentPushButton_4</tabstop>
  <tabstop>TransparentPushButton</tabstop>
  <tabstop>TransparentPushButton_3</tabstop>
  <tabstop>TableWidget_2</tabstop>
  <tabstop>SmoothScrollArea</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>SwitchButton</sender>
   <signal>checkedChanged(bool)</signal>
   <receiver>BodyLabel_5</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SwitchButton</sender>
   <signal>checkedChanged(bool)</signal>
   <receiver>SpinBox_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
